pkts-core/src/main/java/io/pkts/framer/FramingException.java

```diff
@@ -0,0 +1,16 @@
+package io.pkts.framer;
+
+import io.pkts.protocol.Protocol;
+
+public class FramingException extends Exception {
```

Comment:
prefer RuntimeException so will change to that...

Reply:
Sounds good to me!